blocks/footer/footer.css

```diff
@@ -1,18 +1,80 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
+  padding: 2rem;   
+  background-color: #f0f0f0;
   font-size: var(--body-font-size-s);
 }
 
 footer .footer {
   max-width: 1200px;
-  margin: auto;
+  border:0; margin:0 auto; padding:0;
+}
+
+.footer > div > div {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content:center;
+}
+
+footer .footer ul {
+  list-style:none;
+  margin-left: auto;
+  display: flex;
+  overflow: hidden;
+  flex-flow: row wrap;
+  justify-content: flex-end;
+  flex-grow: 1;
+}
+
+footer .footer li {
+  border:0; margin:0; 
+  font-size: 14px;
+  padding: 0 2em;
+  text-align: center;
+  border-right: 1px solid black;
+}
+
+footer .footer li:last-child {
+  border: 0; 
+}
+
+footer .footer  a {
+  font-size: 14px;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;    
+}
+   
+footer .footer a:hover {
+  color: #bd3826;
+}
+
+footer .footer a.active {
+  color: #bd3826;
+}
+
+footer .footer a:focus {
+  color: #bd3826;
```

Comment:
this color is available in the base styles...

Reply:
Ok changed this to the base styles